src/webhooks/models/webhooksFormatter.js

```diff
@@ -126,8 +226,12 @@ module.exports.format = function(format, eventType, jobId, testId, report, addit
         case EVENT_FORMAT_TYPE_JSON: {
             return json(eventType, testId, jobId, report, additionalInfo, options);
         }
+        case EVENT_FORMAT_TYPE_TEAMS: {
+            return teams(eventType, testId, jobId, report, additionalInfo, options);
+        }
         default: {
-            throw new Error(`Unrecognized webhook format: ${format}, available options: ${[EVENT_FORMAT_TYPE_JSON, EVENT_FORMAT_TYPE_SLACK].join()}`);
+            throw new Error(`Unrecognized webhook format: ${format}, available options: ${[EVENT_FORMAT_TYPE_JSON, EVENT_FORMAT_TYPE_SLACK, EVENT_FORMAT_TYPE_TEAMS].join()}`);
```

Comment:
I just noticed we can use the `EVENT_FORMAT_TYPES` instead of specifying all the possibilities in two different sections

so it will turn to 
```js
throw new Error(`Unrecognized webhook format: ${format}, available options: ${EVENT_FORMAT_TYPES.join()}`);
```

Reply:
fixed with commit 6719dd2a4a5e2d612b2793d2536f54afec2b6477